test/integration/command/test_audit.py

```diff
@@ -86,13 +87,13 @@ def test_conan_audit_paths():
 
     # Now some common errors, like rate limited or missing lib, but it should not fail!
     with proxy_response(429, {"error": "Rate limit exceeded"}):
-        tc.run("audit list zlib/1.2.11")
+        tc.run("audit list zlib/1.2.11", assert_error=True)
```

Comment:
Now, `Rate limit exceeded` raises an error, and `Not found` does not; is that expected?

Reply:
I think it makes sense, if you hit the rate limit your cve report will be incomplete.
But having one package not found in the catalog that is something that if you have custom names for your packages may happen, and you have to warn about that but you have a complete report.